deployment/deploy/product.go

```diff
@@ -1,4 +1,4 @@
-package v1_0
+package deploy
```

Comment:
I think we should have different deployer versions considering all version will have selective contracts to deploy. Like - 
```
deployment/v1_0_0/deploy
deployment/v1_6_0/deploy
```
Then v_1_6_0 can internally invoke v1_0_0 deployer if necessary. Otherwise we will compromise on the modularity and every deployment logic across versions will have to be implemented by all versions, Like you had to do for v1_0_0 EVMDeployer